invenio_assets/webpack.py

```diff
@@ -95,6 +95,8 @@ def __html__(self):
         for p in self._paths:
             _, ext = os.path.splitext(p.lower())
             # If we haven't come across the chunk yet, we add it to the output
+            if ext not in request._jinja_webpack_entries:
```

Comment:
@slint Please check this part. When using an invalid file extension it never reaches line 103 below, because it first fails on line 100.